pandas/_libs/util.pxd

```diff
@@ -44,23 +44,50 @@ ctypedef fused numeric:
     cnp.float64_t
 
 
-cdef inline object get_value_at(ndarray arr, object loc):
+cdef inline Py_ssize_t validate_indexer(ndarray arr, object loc) except? -1:
```

Comment:
I don't think you need the question mark in `except?` ? (since -1 can never be actually returned from the function without it being an error)